### PR TITLE
Add support for the "Anywhere on Earth" designation

### PR DESCRIPTION
**Pull request by Tafkas**
Add support for the "Anywhere on Earth" designation #80

### CODE REVIEW

File: maya/core.py [hunk 1]
```diff
@@ -166,6 +166,14 @@ def __dt_to_epoch(dt):
         epoch_start = Datetime(*_EPOCH_START, tzinfo=pytz.timezone('UTC'))
         return (dt - epoch_start).total_seconds()
 
+    @property
+    def aoe(self):
+        """
+        Return True if the date has not passed anywhere on earth.
+        See also: http://www.ieee802.org/16/aoe.html
+        """
+        return when('tomorrow noon') > self > when('yesterday noon')
```
Comment (davidism) [inline maya/core.py]: Should one of these be inclusive? Python likes to make the lower bound inclusive, so `>= yesterday noon`.

File: maya/core.py [hunk 2]
```diff
@@ -417,6 +425,14 @@ def intersects(self, maya_interval):
     def midpoint(self):
         return self.start.add(seconds=(self.duration / 2))
 
+    @property
+    def aoe(self):
+        """
+        Return True if the interval has not passed anywhere on earth.
+        See also: http://www.ieee802.org/16/aoe.html
+        """
+        return self.start.aoe or self.end.aoe
```
Comment (davidism) [inline maya/core.py]: This isn't correct. You also need to check if the inverval is active, not just the endpoints. `self.start <= now < self.end`. (Adjust inclusivity as needed.)

File: maya/core.py (hunk 1 above)
Comment (davidism) [inline maya/core.py]: Also, `<` always looks more natural to me, but that's just a style thing.

**Comment by kennethreitz** [comment]
Sorry for not leaving a comment — I closed this due to https://github.com/kennethreitz/maya/issues/80#issuecomment-306004204

**Pull request closed without merging**